src/main/resources/i18n/messages.properties

```diff
@@ -632,6 +632,10 @@ ranked1v1.showMaps=Show ladder maps
 game.create.onlyFriends=Only show to friends
 settings.language.helpUs.text=Downlord's FAF Client's localization is a community effort. If you want to see the client translated into another language we'd love your help.
 settings.language.helpUs.button=Help us translate
+replay.notFoundTitle=Oh oh. No replay for this found.
```

Comment:
I'm sure we can come up with a better title than this. Also, titles don't end with periods.